packages/xchain-ethereum/src/utils.ts

```diff
@@ -92,7 +92,7 @@ export const validateAddress = (address: Address): boolean => {
  */
 export const getTokenAddress = (asset: Asset | null): string | null => {
```

Comment:
Can we change `(asset: Asset | null)` to `(asset: Asset)` for parameters of  `getTokenAddress`? _It will be a breaking change then (just for CHANGELOG) ..._